### PR TITLE
New page on why flowforge

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Created a new page with the content described in the issue

## Related Issue(s)

#1015

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
Hi @iskerrett here's the why ff page. We still have some work to do, but only small tasks:

- [ ] Could you please replace the placeholder text I used with the actual description?
- [ ] We need to decide where we're going to include this page in the nav bar.
- [ ] And, when we have the videos on youtube, I'll replace the ones we currently have.

By the way, are we still waiting for more comments on the video? Or should I work in the other 3 we've prepared?

File: src/_data/reasons.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[{
+    "title": "Hosted Node-RED - Easy to get started, easier to maintain",
+    "content": "FlowForge Cloud takes care of running Node-RED for you. No more messing around with Docker containers, upgrading to new releases, keeping your Node-RED running, we take care of it for you.",
```
Comment (iskerrett) [inline src/_data/reasons.json:3]:
```suggestion
    "content": "FlowFuse Cloud takes care of running Node-RED for you. No more messing around with Docker containers, upgrading to new releases, keeping your Node-RED running, we take care of it for you.",
```

File: src/_data/reasons.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "videoid": "K4xw09DbSdI"
+  },
+  {
+    "title": "Multiple Node-RED Instances from one place",
+    "content": "FlowForge Cloud takes care of running Node-RED for you. No more messing around with Docker containers, upgrading to new releases, keeping your Node-RED running, we take care of it for you.",
```
Comment (iskerrett) [inline src/_data/reasons.json:8]:
```suggestion
    "content": "FlowFuse allows you to create multiple Node-RED instances and manage them in one place. Gone are the days of trying to coordinate Node-RED installations, or multitude of tabs. Finally Node-RED allows for serious application development.
",
```

File: src/why-flowfuse.njk [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: layouts/page.njk
+title: Why FlowFuse?
+description:
+    <p>Lorem ipsum dolor sit amet consectetur. Est id morbi nascetur tortor id donec tellus arcu enim. Leo dolor montes sapien quam. Pellentesque a purus parturient egestas quis nibh est.</p>
```
Comment (iskerrett) [inline src/why-flowfuse.njk:5]:
```suggestion
    <p>Node-RED is a great low-code visual programming tool. FlowFuse adds to standard Node-RED to make it even better.</p>
```

**Comment by iskerrett** [comment]
> Hi @iskerrett here's the why ff page. We still have some work to do, but only small tasks:
> 
> * [x]  Could you please replace the placeholder text I used with the actual description?
> * [ ]  We need to decide where we're going to include this page in the nav bar.

I suggest we add it to the product menu. I would make it the first item.

> * [ ]  And, when we have the videos on youtube, I'll replace the ones we currently have.
> 
> By the way, are we still waiting for more comments on the video? Or should I work in the other 3 we've prepared?

We are still waiting for feedback. I will remind people. :-)

**Comment by iskerrett** [comment]
btw, I really like this new page :-)

**Comment by iskerrett** [comment]
I think we should also include #5 and #6. I know we won't have a video but we can add those later.

**Comment by Yndira-FlowForge** [comment]
> I suggest we add it to the product menu. I would make it the first item.

Done

> I think we should also include 5 and 6. I know we won't have a video but we can add those later.

And done!

# :tada:

File: src/product/why-flowfuse.njk [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+layout: layouts/page.njk
+title: Why FlowFuse?
+description:
+    <p>Node-RED is a great low-code visual programming tool. FlowFuse adds to standard Node-RED to make it even better.</p>
```
Comment (iskerrett) [inline src/product/why-flowfuse.njk:5]:
```suggestion
    <p>Node-RED is a great low-code visual programming tool. Here are 6 things FlowFuse adds to standard Node-RED to make it even better.</p>
```

**Comment by iskerrett** [comment]
> Hi @iskerrett, I've added the intros to all the videos. If you find them okay

Intros look good. I will upload the videos to YouTube.

**Pull request merged**